docs/trestle_author.md

```diff
@@ -18,347 +27,573 @@ While trestle provides editing support for OSCAL there is an unfortunate truth t
+    This will create a sample document in: `TRESTLE_ROOT/my_task_name/my_task_name_000.md`.
+    Once created this document will be a complete copy of the template, however you can modify this file with your own content.
 
-`trestle author docs setup -tn my_task_name` Create the necessary directory structures for running governed docs validation.
-A template file will be created in `TRESTLE_ROOT/.trestle/author/my_task_name/template.md` and be applied to all markdown files here: `TRESTLE_ROOT/my_task_name/*.md`.
+    #### Extra options
 
-`trestle author docs create-sample -tn my_task_name` Creates a sample file in `TRESTLE_ROOT/my_task_name/`
+    - This command has no extra options
 
-`trestle author docs template-validate -tn my_task_name` Ensures that the markdown is parseable. If `--governed-heading 'heading name'` is passed it ensures that the required heading exists.
+    ### Validating the template
 
-`trestle author docs validate -tn my_task_name` validates the markdown, optionally with a `--governed-heading` or `-hv` yaml header based on this `TRESTLE_ROOT/.trestle/author/my_task_name/template.md` template to all markdown files here: `TRESTLE_ROOT/my_task_name/*.md`.
+    To ensure that the markdown in the template is parseable run:
 
-`trestle author docs validate -tn my_task_name -ig ^_.*` will validate all files except folders and files that start with underscore `_`. Use option `-ig` or `--ignore` when you would like to ignore any folders or files that match given regular expression.
+    > trestle author docs template-validate -tn my_task_name
 
-### Extra options
+    #### Extra options
 
-#### recursive (`-r`, `--recurse`)
+    - If `--governed-heading 'heading name'` (`-gh`) is passed it ensures that the required heading exists.
+    - If `--header-validate` (`-hv`) is passed the header will be validated as well.
+    - If `--header-only-validate` (`-hov`) only the header and NOT the body will be validated
 
-By default `author docs` only indexes a flat directory. The recursive option allows the markdown files to be nested in sub-directories.
+    ### Validating the documents against the template
 
-#### Header only validation (`-hov`, `--header-only-validate`)
+    To validate the documents against a template run:
 
-Turns off the validation of the structure of the document and only validates the yaml header structure.
+    > trestle author docs validate -tn my_task_name
 
-#### Ignore files or folders from validation (`-ig`, `--ignore`)
+    This will take this `TRESTLE_ROOT/.trestle/author/my_task_name/template.md` template and validate all markdown files here: `TRESTLE_ROOT/my_task_name/*.md`.
 
-Provide regular expression after this flag to ignore folders and files which names match this regular expression during **validation only**.
+    This will validate that markdown body in the document is valid against the template. Please note that by default the header will **not** be validated.
```

Comment:
Running the command will validate that...

Reply:
Done